.github/workflows/enarx-rust.yml

```diff
@@ -0,0 +1,25 @@
+name: Enarx Rust Tests
+# Automated testing specific to Enarx's Rust repositories.
+
+on: [pull_request]
+
+jobs:
+  # Ensure all Rust code is properly formatted with `rustfmt`.
+  rust-formatting:
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: quay.io/enarx/fedora-test
```

Reply:
What is the purpose of the `ubuntu-latest` since we're using our own container?

Reply:
All Actions must specify a runner -- either Github- or self-hosted. We aren't self hosting, and the available Github ones are [here](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on). "Ubuntu" is basically "Linux" in this case.

I think it's more accurate to say our Fedora container is running via Docker on Github-hosted Ubuntu.